benchmarks/python/test_dropout_rmsnorm_bwd.py

```diff
@@ -128,10 +158,40 @@ def test_rmsnorm_bwd_benchmark(
         )
         eager_output.backward(grads.to(torch.double))
         fd.validate(
-            [inputs, dropout_mask, rms_eps, grads, weights], [inputs.grad, weights.grad]
+            [input1, input2, dropout_mask, rms_eps, grads, weights],
+            [input1.grad, input2.grad, weights.grad],
         )
 
     if not disable_benchmarking:
         run_benchmark(
-            benchmark, fd.execute, [inputs, dropout_mask, rms_eps, grads, weights]
+            benchmark,
+            fd.execute,
+            [input1, input2, dropout_mask, rms_eps, grads, weights],
         )
+
+
+@pytest.mark.parametrize("compile", [False, True], ids=["eager", "compile"])
+@pytest.mark.parametrize("size", generate_input_sizes(dims=2))
+@pytest.mark.parametrize("dtype", FLOAT_DTYPES)
+def test_dropout_rmsnorm_bwd_baseline_benchmark(
+    benchmark,
+    size: tuple,
+    dtype: torch.dtype,
+    compile: bool,
+):
+    clear_cuda_cache()
+    dropout_p = 0.2
+    input1 = torch.randn(size, device="cuda", dtype=dtype, requires_grad=True)
+    input2 = torch.randn(size, device="cuda", dtype=dtype, requires_grad=True)
+    grads = torch.randn(size, device="cuda", dtype=dtype)
+    weights = torch.randn(size[1], device="cuda", dtype=dtype)
+
+    x = input2 + torch.nn.functional.dropout(input1, p=dropout_p)
+    output = weights * torch.rsqrt(x.pow(2).mean(-1, keepdim=True) + 1e-5)
```

Comment:
Since only `output` and `grads` are needed, why not just generate them with `torch.randn` instead of the actual forward?

Reply:
I don't see how that would still have the same performance. The backward pass will look different in this case since there wouldn't be any actual computation graph, most likely producing different performance numbers.

Reply:
Oh of course. I missed that this uses `unary_bwd_torch` instead of a direct implementation of the backward. Nevermind!